src/mumble/AudioOutput.cpp

```diff
@@ -360,7 +360,9 @@ void AudioOutput::initializeMixer(const unsigned int *chanmasks, bool forceheadp
 }
 
 bool AudioOutput::mix(void *outbuff, unsigned int nsamp) {
+	// A list of all users that have audio to contribute
```

Comment:
What about removing "all" to match the the comment below?

Reply:
Done